### ISSUE TITLE
Will callsign need to change when 1B is deployed ?

### ISSUE THREAD

**Issue by fraab** [FOSSASystems/FOSSA-Comms#6]
Currently the callsign in all programs is set to FOSSASAT-1.  Will it change to FOSSASAT-1B when the new satellite is deployed?

**Comment by Gipsonek** [comment]
That's correct, packets sent to/from any given satellite will always have the correct callsign - "FOSSASAT-1B" for FOSSASAT-1B, "FOSSASAT-2" for FOSSASAT-2 etc.

The examples here only show the basic usage - IMO it doesn't make much sense to have separate examples for all the satellites.
